pkg/cmd/cli/cmd/whoami.go

```diff
@@ -50,3 +54,39 @@ func (o WhoAmIOptions) WhoAmI() (*userapi.User, error) {
 
 	return me, err
 }
+
+func RunWhoAmI(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []string, o *WhoAmIOptions) error {
+	if kcmdutil.GetFlagBool(cmd, "token") {
+		cfg, err := f.OpenShiftClientConfig.ClientConfig()
+		if err != nil {
+			return err
+		}
+		if len(cfg.BearerToken) == 0 {
+			return fmt.Errorf("no token is currently in use for this session")
+		}
+		fmt.Fprintf(out, "%s\n", cfg.BearerToken)
+		return nil
+	}
+	if kcmdutil.GetFlagBool(cmd, "context") {
```

Comment:
So that you can script `oc config use-context` across multiple command lines, or print a value.  We could put it on config but "whoami" is close enough to the "id" concept in unix that it doesn't seem meaningful to move it.